ste/performanceAdvisor.go

```diff
@@ -108,7 +108,7 @@ type PerformanceAdvisor struct {
 	serverBusyPercentageOther      float32
 	iops                           int
 	mbps                           int64
-	capMbps                        int64 // 0 if no cap
+	capMbps                        float64 // 0 if no cap
```

Comment:
This change will break the call to addAdvice that uses this variable (because it currently has the %d format verb).  Can you please adjust the format verb, and test benchmark mode with integer and fractional values of capMbps.

Reply:
Yep noted on this, thanks for the heads-up!